packages/jupyter-ai/jupyter_ai/config_manager.py

```diff
@@ -456,7 +471,8 @@ def _provider_params(self, key, listing):
             authn_fields[keyword_param] = config.api_keys[key_name]
 
         return {
-            "model_id": lid,
+            "model_id": model_id,
+            **fields,
```

Comment:
I guess this is the crucial line, right?

Otherwise, I appreciate renaming of the variables it makes it much more readable than the acronyms that were previously used.

Reply:
Glad you find these naming conventions more reasonable! I'm tracking suggestions for new naming conventions here: #1123.